tests/Routes/Wiki/CreateTest.php

```diff
@@ -27,9 +28,17 @@ class CreateTest extends TestCase
     /**
 	 * @dataProvider createProvider
 	 */
-    public function testWikiCreateDispatchesSomeJobs( $elasticSearchEnabledForNewWikis )
+    public function testWikiCreateDispatchesSomeJobs( $elasticSearchConfig )
     {
-        Config::set('wbstack.elasticsearch_enabled_by_default', $elasticSearchEnabledForNewWikis );
+        $enabledForNewWikis = $elasticSearchConfig[ 'enabledForNewWikis' ];
+        $clusterWithoutSharedIndex = $elasticSearchConfig[ 'clusterWithoutSharedIndex' ] ?? null;
+        $sharedIndexHost = $elasticSearchConfig[ 'sharedIndexHost' ] ?? null;
+        $sharedIndexPrefix = $elasticSearchConfig[ 'sharedIndexPrefix' ] ?? null;
+
+        Config::set( 'wbstack.elasticsearch_enabled_by_default', $enabledForNewWikis );
```

Comment:
Question: will other tests see the changes made here or will Laravel give us a fresh config on each test?